# -----------------------------------------------------------
# Fix the function
# 
# I created this function to add five to any number that was passed in to it and return the new value. It doesn't throw any errors but it returns the 
# wrong number.
# 
# Can you help me fix the function?
# -----------------------------------------------------------

def add_five(num):
    total = num + 5
    return total

# or

def add_five(num):
    return num + 5

# or

add_five = lambda num: num + 5

# -----------------------------------------------------------
# License
# Tasks are the property of Codewars (https://www.codewars.com/) 
# and users of this resource.
# 
# All solution code in this repository 
# is the personal property of Vladimir Rukavishnikov
# (vladimirrukavishnikovmail@gmail.com).
# 
# Copyright (C) 2022 Vladimir Rukavishnikov
# 
# This file is part of the HungryVovka/Codewars-Python
# (https://github.com/HungryVovka/Codewars-Python)
# 
# License is GNU General Public License v3.0
# (https://github.com/HungryVovka/Codewars-Python/blob/main/LICENSE.md)
# 
# You should have received a copy of the GNU General Public License v3.0
# along with this code. If not, see http://www.gnu.org/licenses/
# -----------------------------------------------------------